arch/arm/src/armv8-r/sctlr.h

```diff
@@ -32,8 +32,9 @@
  * Included Files
  ****************************************************************************/
 
+#include <arch/irq.h>
```

Comment:
don't need

Reply:
sctlr.h requires the relevant definition of cp15